src/modules/dashboard/components/ColonyHome/ColonyUnclaimedTransfers/ColonyUnclaimedTransfers.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useCallback } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import {
+  useColonyTransfersQuery,
+  Colony,
+  useTokenQuery,
+  useLoggedInUser,
+} from '~data/index';
+
+import { ActionButton } from '~core/Button';
+import Heading from '~core/Heading';
+import { MiniSpinnerLoader } from '~core/Preloaders';
+import NavLink from '~core/NavLink';
+import Numeral from '~core/Numeral';
+import Tooltip from '~core/Popover';
+
+import { ActionTypes } from '~redux/index';
+import { mergePayload } from '~utils/actions';
+import { checkIfNetworkIsAllowed } from '~utils/networks';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+
+import styles from './ColonyUnclaimedTransfers.css';
+
+const displayName = 'dashboard.ColonyUnclaimedTransfers';
+
+interface Props {
+  colony: Colony;
+}
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.ColonyUnclaimedTransfers.title',
```

Comment:
The above then also applies to message descriptor id's

```suggestion
    id: 'dashboard.ColonyHome.ColonyUnclaimedTransfers.title',
```